docs/installation/server-config/config-reference.rst

```diff
@@ -160,18 +160,20 @@ will be prefixed as ``_confluent-ksql-default_`` (e.g. ``_command_topic`` become
 .. _ksql-sink-partitions:
 
 --------------------
-ksql.sink.partitions
+ksql.sink.partitions (Depricated)
 --------------------
 
 The default number of partitions for the topics created by KSQL. The default is four.
+This property has been depricated since 5.3 release. For more info see :ref:`KSQL Syntax Reference <ksql_syntax_reference>`.
```

Comment:
Can we provide a more specific link than the general syntax reference link? Ideally we'd link directly to the "partitions" row in the relevant table. If that's not possible, we can link to the table itself and call out that the user should be looking at the "partitions" row.

Same comment for "replicas" below.

Reply:
The table rows could not be referenced. I referenced the section and mentioned the WITH clause table instead.